src/main/java/com/xpanxion/java/assignments/student1/Worker1.java

```diff
@@ -33,4 +37,23 @@ public void ex3() {
         var result = products.stream().filter(p -> p.getPrice() >= 10.0);
         System.out.println(result.collect(Collectors.toList()));
     }
+
+    public void ex4() {
+        var products = DataAccess.getProducts();
+        Optional<Float> total;
+        NumberFormat fomatter = NumberFormat.getCurrencyInstance();
+        String moneyString;
+
+        Map<Integer, Department> departmentMap = DataAccess.getDepartments().stream()
```

Comment:
Let's see if we can simplify this.  I don't think there's a good reason to convert the department list to a Map.